content/docs/tasks/security/authz-tcp/index.md

```diff
@@ -0,0 +1,231 @@
+
+## Installing and configuring a TCP service
+
+By default, the [Bookinfo](/docs/examples/bookinfo/) example application only includes HTTP services.
+To show how Istio handles the authorization of TCP services, we must update the application to use a
+TCP service. Follow this procedure to deploy the Bookinfo example app and update its `ratings` service
+to the `v2` version, which talks to a MongoDB backend using TCP:
+
+1. Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application:
+
+    After it has been deployed, point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). You should see:
+
+        * The "Book Details" section in the lower left part of the page, including type, pages, publisher, etc.
+        * The "Book Reviews" section in the lower right part of the page.
+
+    If you refresh the page several times, you should see different versions of reviews shown in the
+    product page, presented in a round robin style (red stars, black stars, no stars)
+
+1. Install `v2` of the `ratings` service with service account `bookinfo-ratings-v2`:
+
+    In this task, we will enable access control using service accounts, which are cryptographically
+    authenticated in the mesh. In order to give different microservices different access privileges,
+    we will create the `v2` of the `ratings` with service account `bookinfo-ratings-v2`. Other services
+    will have a default service account `default`.
+
+    If you are using a cluster with automatic sidecar injection enabled,
+    simply deploy the service using `kubectl`:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/rbac/ratings-v2-add-serviceaccount.yaml@
+    serviceaccount "bookinfo-ratings-v2" created
+    deployment "ratings-v2" configured
+    {{< /text >}}
+
+    If you are using manual sidecar injection, use the following command instead:
+
+    {{< text bash >}}
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/rbac/ratings-v2-add-serviceaccount.yaml@)
+    serviceaccount "bookinfo-ratings-v2" created
+    deployment "ratings-v2" configured
+    {{< /text >}}
+
+1. Update the Bookinfo sample to use the new version of `ratings`:
+
+    The Bookinfo sample deploys multiple versions of each microservice, so you will start by creating
+    destination rules that define the service subsets corresponding to each version, and the load
+    balancing policy for each subset.
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/networking/destination-rule-all-mtls.yaml@
+    destinationrule "productpage" created
+    destinationrule "reviews" created
+    destinationrule "ratings" created
+    destinationrule "details" created
+    {{< /text >}}
+
+    Since the subset referenced in virtual services rely on the destination rules,
+    wait a few seconds for destination rules to propagate before adding virtual services that refer
+    to these subsets.
```

Comment:
This entire text causes a lot of confusion, IMO. It also exposes a major issue I have with this document: The inconsistent use of terms for the services in the Bookinfo app. Please pick one and use it consistently throughout. The options are: service (which is the most employed), microservice (I would not recommend this one), or virtual service (my preferred option). 
Specifically, I would get rid of the text here. I would then add a conditional to the numbered step. See my comment on line 87

Reply:
virtual services is not the same to the general meaning services, it's the VirtualService custom resource https://istio.io/docs/reference/config/istio.networking.v1alpha3/#VirtualService

Reply:
Small tip: don't resolve conversations you didn't start. :wink: 
If you are being specific because the steps require the CRD of kind `VirtualService`,  then you should replace all instances of the term "service" in this document with the term "virtual service".

Reply:
I've edited the text in case you do want to include it:
    Since the subset referenced in the virtual services relies on the destination rules,
    wait a few seconds for the destination rules to propagate before adding the virtual services.

Reply:
The virtual services here refers to the specific `VirtualServices` added in the following rule `samples/bookinfo/networking/virtual-service-ratings-db.yaml`. And has nothing to do with other services used in this document. So probably change it to `virtual service rules` here to make it clear it's actually referring to the rules below.